test/unit/controllers/InvitationController.test.js

```diff
@@ -76,15 +73,12 @@ describe('InvitationController', () => {
       })
     })
 
-    it('sends invitations', function () {
-      this.timeout(10000)
+    it('sends invitations', () => {
       _.extend(req.params, {communityId: community.id, emails: 'foo@bar.com, bar@baz.com'})
 
       return InvitationController.create(req, res)
       .then(() => {
         expect(Email.sendInvitation).to.have.been.called.exactly(2)
-        expect(apiResult).to.exist
```

Comment:
That is fair enough, the reason we went with this solution was because that section of the code that handles this is unit tested elsewhere. I am looking into another way of getting this test working for us now without mocking.